Source/WebCore/html/RadioInputType.cpp

```diff
@@ -215,10 +216,10 @@ void RadioInputType::willDispatchClick(InputElementClickState& state)
     // Therefore if nothing is currently selected, we won't allow the upcoming action to be "undone", since
     // we want some object in the radio group to actually get selected.
 
-    state.checked = element()->checked();
-    state.checkedRadioButton = element()->checkedRadioButtonForGroup();
+    state.checked = protectedElement()->checked();
```

Comment:
Ditto about storing it in a local variable instead of keep calling protectedElement.